### PR TITLE
[DE] Update `Help centre/Store`

### PR DESCRIPTION
**Pull request by The-Last-Cookie**
## Self-check

- [x] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)
- [x] *(translations only)* The changes are reviewed on GitHub [by a fluent speaker](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#review)

### CODE REVIEW

File: wiki/Help_centre/Store/de.md [hunk 1]
```diff
@@ -2,128 +2,54 @@
...
+Kontaktiere gerne [merch@ppy.sh](mailto:merch@ppy.sh), wenn du noch weitere Fragen hast. Beachte jedoch, dass nachdem wir ein Produkt versandt haben, wir genau so viel über den Lieferstatus wissen wie du.
 
-Bitte kontaktiere [store@ppy.sh](mailto:store@ppy.sh) für weitere Rückfragen.
+## Ich benötige Hilfe mit einem bestimmten Produkt!
 
-#### LEDs auf Korrosion überprüfen {id=osu!keyboard-corrosion}
+### osu! keycaps {id=osu!-keycaps}
 
-**Reibe die Basis der LED-Anschlüsse mit einem kleinen Quadrat aus Alufolie ab, um die meisten Korrosionsrückstände zu entfernen.**
+[osu! keycaps](https://www.thekapco.com/products/osu-pippi-keycap-set) (Tastenkappen) werden von The KapCo hergestellt und vertrieben, wir empfehlen bei Problemen mit Keycaps also [mit deren Kundensupport](https://www.thekapco.com/pages/faqs) zu reden.
```
Comment (FreezyLemon) [inline wiki/Help_centre/Store/de.md:43]:
```suggestion
[osu! keycaps](https://www.thekapco.com/products/osu-pippi-keycap-set) (Tastenkappen) werden von The KapCo hergestellt und vertrieben. Deshalb empfehlen wir bei Problemen mit Keycaps, sich an [ihren Kundensupport](https://www.thekapco.com/pages/faqs) zu wenden.
```

File: wiki/Help_centre/Store/de.md [hunk 2]
```diff
@@ -2,128 +2,54 @@
...
 
-Korrosionsrückstände erscheinen in der Regel schwärzlich-grau oder können als seltsame Flecken auf dem Metall erscheinen.
+Sollte der Kundensupport keine zufriedenstellende Antwort liefern, kontaktiere [merch@ppy.sh](mailto:merch@ppy.sh) und wir werden sehen, was wir tun können.
 
-Das Entfernen dieser Rückstände bringt deine LED wahrscheinlich wieder zum Laufen. Wenn das der Fall ist, weißt du, wie du es beim nächsten Mal beheben kannst!
+### osu! deskmat {id=osu!-deskmat}
 
-## osu!tablet {id=osu-tablet}
+Während viele osu!-Produkte von unserem Warenlager in den Vereinigten Staaten versandt werden, werden [osu! deskmats](https://osu.ppy.sh/store/products/1347) (Tischunterlagen) von verschiedenen Verteilerzentren im Vereinigten Königreich, in den Vereinigten Staaten und in Australien vertrieben. Das heißt, wir haben nicht über jeden Artikel, der geliefert wird, einen genauen Überblick.
```
Comment (FreezyLemon) [inline wiki/Help_centre/Store/de.md:49]:
```suggestion
Während viele osu!-Produkte von unserem Warenlager in den Vereinigten Staaten versandt werden, werden [osu! deskmats](https://osu.ppy.sh/store/products/1347) (Tischunterlagen) von verschiedenen Verteilerzentren im Vereinigten Königreich, in den Vereinigten Staaten und in Australien versandt. Das heißt, wir haben nicht über jeden Artikel, der geliefert wird, einen genauen Überblick.
```

File: wiki/Help_centre/Store/de.md [hunk 3]
```diff
@@ -2,128 +2,54 @@
...
+Dieser Artikel bezieht sich nur auf Produkte, die im [osu!store](https://osu.ppy.sh/store/listing) erworben werden können.
 
-## osu!keyboard {id=osu!keyboard}
+## Ich möchte meine Anschrift ändern! {id=address}
 
-### Wie konfiguriere ich das osu!keyboard? {id=osu!keyboard-setup}
+Sende eine E-Mail mit der Bestellnummer und der neuen Adresse an [merch@ppy.sh](mailto:merch@ppy.sh).
 
-**Du kannst das osu!keyboard-Konfigurationsprogramm verwenden, das auf [puush](https://puu.sh/l6urN/4b6bc800f2.zip) zur Verfügung steht.**
+Wenn deine Bestellung noch nicht versandt wurde, aktualisieren wir die Adresse und senden die Bestellung an die neue Anschrift. Wenn die Bestellung bereits unterwegs ist, können keine Änderungen mehr vorgenommen werden.
```
Comment (FreezyLemon) [inline wiki/Help_centre/Store/de.md:27]: Bit less formal maybe

```suggestion
Wenn deine Bestellung noch nicht versandt wurde, aktualisieren wir die Adresse und senden die Bestellung an die neue Anschrift. Wenn die Bestellung bereits unterwegs ist, können wir leider nicht mehr viel machen.
```

.. is `machen` correct there? maybe `tun`

File: wiki/Help_centre/Store/de.md [hunk 4]
```diff
@@ -2,128 +2,54 @@
...
 
-### Die LEDs auf meinem osu! nono keyboard funktionieren nicht! {id=osu!keyboard-leds}
+Gelegentlich bleiben Pakete im Versandsystem stecken. Wenn deine Bestellung irgendwo in den Vereinigten Staaten festsitzt, kann sie möglicherweise mit [USPS](https://faq.usps.com/s/article/USPS-Tracking-The-Basics) nachverfolgt werden. Dazu kann normalerweise die zuständige Poststelle kontaktiert werden. Wenn die Bestellung außerhalb der Vereinigten Staaten festsitzt, kontaktiere den Postdienst deines Landes für weitere Hilfe.
 
-**Dafür kann es mehrere Gründe geben — Korrosion zwischen den LEDs und dem Mainboard oder in bestimmten Fällen, insbesondere mit früheren Modellen, defekte LEDs.**
+Kontaktiere gerne [merch@ppy.sh](mailto:merch@ppy.sh), wenn du noch weitere Fragen hast. Beachte jedoch, dass nachdem wir ein Produkt versandt haben, wir genau so viel über den Lieferstatus wissen wie du.
```
Comment (FreezyLemon) [inline wiki/Help_centre/Store/de.md:37]:
```suggestion
Kontaktiere gerne [merch@ppy.sh](mailto:merch@ppy.sh), wenn du noch weitere Fragen hast. Beachte jedoch, dass wir genau so viel über den Lieferstatus wissen wie du, nachdem wir ein Produkt versandt haben.
```

**Review by FreezyLemon — CHANGES_REQUESTED** [review]
I think `vertrieben` -> `versandt` should be changed since there's a clear difference in meaning. The rest are just suggestions

File: wiki/Help_centre/Store/de.md (hunk 3 above)
Comment (The-Last-Cookie) [reply wiki/Help_centre/Store/de.md:27]: According to [this article](https://dein-sprachcoach.de/unterschied-zwischen-machen-und-tun/), `tun` is used for formal texts and `machen` in informal conversations. `Tun` is also used for emphasising specific actions.

You can say both, though my gut feeling tells me that `etwas für jemanden tun` is more common then `etwas für jemanden machen`.

File: wiki/Help_centre/Store/de.md [hunk 5]
```diff
@@ -2,128 +2,54 @@
...
 
-Der Rest sollte selbsterklärend sein.
+Beim Kauf eines Produkts aus dem osu!store erhältst du eine E-Mail mit einer Sendungsverfolgung, sobald das Produkt versandt wurde.
 
-Wenn du weitere Probleme hast, sende bitte ein Ticket an [support@ppy.sh](mailto:support@ppy.sh), indem du dein Problem näher erläuterst.
+Wenn deine Bestellung in deinem Land ankam, aber noch nicht geliefert wurde, wird sie möglicherweise bei deinem lokalen Postamt aufbewahrt. Wende dich an die Poststelle und nenne deine Sendungsnummer für weitere Details! Das ist für internationale Käufe, bei denen Zollgebühren oder andere Abgaben anfallen, manchmal erforderlich.
 
-### Die LEDs auf meinem osu! nono keyboard funktionieren nicht! {id=osu!keyboard-leds}
+Gelegentlich bleiben Pakete im Versandsystem stecken. Wenn deine Bestellung irgendwo in den Vereinigten Staaten festsitzt, kann sie möglicherweise mit [USPS](https://faq.usps.com/s/article/USPS-Tracking-The-Basics) nachverfolgt werden. Dazu kann normalerweise die zuständige Poststelle kontaktiert werden. Wenn die Bestellung außerhalb der Vereinigten Staaten festsitzt, kontaktiere den Postdienst deines Landes für weitere Hilfe.
```
[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/Help_centre/Store/de.md:35]: I wondered as well while translating.

A German postal service will only make tracking available after receiving the packet, don't they?

Including a link to DHL or something (and mention USPS instead on the side) sounds like a good idea to me.

File: wiki/Help_centre/Store/de.md (hunk 3 above)
Comment (FreezyLemon) [reply wiki/Help_centre/Store/de.md:27]: I was proposing the less formal variant on purpose after looking at the English text. Both are fine, but the current version seems a bit stiff.

File: wiki/Help_centre/Store/de.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (FreezyLemon) [reply wiki/Help_centre/Store/de.md:35]: I don't know enough about the osu store to say either way. What I *can* tell you is that DHL does have tracking for anything "Päckchen" and above (Paket, Speditionslieferung) but not for normal Briefe or Postkarten. So unless the store products are sent in a normal letter envelope, there should be some kind of tracking. That said, I have no clue which logistics company/Spedition is used. There's also Hermes, DPD, GLS, UPS and some others.

[the remark this answers is not in the document]
Comment (FreezyLemon) [reply wiki/Help_centre/Store/de.md:35]: Note that this paragraph is not about normal tracking, it's about packages getting stuck somewhere along the way. Jason is right that you'd use the tracking website/service provided by the USPS even if local handling is done by German Post or DHL. But you wouldn't contact a USPS office if your package gets stuck somewhere in a DHL sorting center, I think.

It's probably not that deep. Even the current text is *fine*, as it does direct you to a national post service

[the remark this answers is not in the document]
Comment (The-Last-Cookie) [reply wiki/Help_centre/Store/de.md:35]: I will leave the text as-is then if you think it's okay.

We can adjust anytime if people complain later on.

**Comment by The-Last-Cookie** [comment]
I overwrote last commit's message to be more descriptive.

**Review by FreezyLemon — APPROVED** [review]

**Review by Hiviexd — APPROVED** [review]

**Pull request merged**